README.md

```diff
@@ -8,41 +8,123 @@
+
+Important note: It is **not recommended or supported** to use both the raw and transactional APIs on the same database.
+
+### Code examples
+
+Raw mode:
+
+```rust
+let config = Config::new(vec!["127.0.0.1:2379"]);
+let client = RawClient::new(config).await?;
+client.put("key".to_owned(), "value".to_owned()).await;
+let value = client.get("key".to_owned()).await;
+```
+
+Transactional mode:
+
+```rust
+let config = Config::new(vec!["127.0.0.1:2379"]);
+let txn_client = TransactionClient::new(config).await?;
+let mut txn = txn_client.begin().await?;
+txn.put("key".to_owned(), "value".to_owned()).await?;
+let value = txn.get("key".to_owned()).await;
+txn.commit().await?;
+```
+
+There are some [examples](examples) which show how to use the client in a Rust program.
+
+### API
+
+#### Raw requests
+
+| Request        | Main parameter type | Successful result type | Noteworthy Behavior                           |
+| -------------- | ------------------- | ---------------------- | --------------------------------------------- |
+| `put`          | `KvPair`            | `()`                   |                                               |
+| `get`          | `Key`               | `Option<Value>`        |                                               |
+| `delete`       | `Key`               | `()`                   |                                               |
+| `scan`         | `BoundRange`        | `Vec<KvPair>`          |                                               |
+| `batch_put`    | `Iter<KvPair>`      | `()`                   |                                               |
+| `batch_get`    | `Iter<Key>`         | `Vec<KvPair>`          | Skip non-existent keys; Does not retain order |
+| `batch_delete` | `Iter<Key>`         | `()`                   |                                               |
+| `delete_range` | `BoundRange`        | `()`                   |                                               |
+
+#### Transactional requests
+
+| Request     | Main parameter type | Successful result type | Noteworthy Behavior                           |
+| ----------- | ------------------- | ---------------------- | --------------------------------------------- |
+| `put`       | `KvPair`            | `()`                   |                                               |
+| `get`       | `Key`               | `Option<value>`        |                                               |
+| `delete`    | `Key`               | `()`                   |                                               |
+| `scan`      | `BoundRange`        | `Iter<KvPair>`         |                                               |
+| `batch_get` | `Iter<Key>`         | `Iter<KvPair>`         | Skip non-existent keys; Does not retain order |
+| `lock_keys` | `KvPair`            | `()`                   |                                               |
+
+For detailed behavior of each reqeust, please refer to the [doc](#Access-the-documentation).

```

Comment:
```suggestion
For detailed behavior of each request, please refer to the [doc](#Access-the-documentation).

```

Reply:
`batch_scan` is a raw request. So TiDB should not have this problem.